test/cleanBoard.test.js

```diff
@@ -0,0 +1,8 @@
+import { cleanBoard } from '../src/scripts/cleanBoard';
+
+describe('should reset the board', () => {
+  const board = cleanBoard([null, null, null, null, "X", null, "O", "X", null]);
+  it('should return an array of 9 nulls', () => {
```

Comment:
- [ ] [OPTIONAL] If you want to check if the array has a length of 9 and is filled with null you re checking two things at the same time. You can have two tests them, One for the length, and another for the contents.